src/sql/workbench/services/connection/browser/connectionWidget.ts

```diff
@@ -757,6 +761,12 @@ export class ConnectionWidget extends lifecycle.Disposable {
 		if (!selectedAccount && selectFirstByDefault && this._azureAccountList.length > 0) {
 			selectedAccount = this._azureAccountList[0];
 		}
+
+		if (this.authenticationType === AuthenticationType.AzureMFAAndUser) {
+			// Populate username as 'email' of selected azure account in dropdown (Ask from MySQL)
+			this._userNameInputBox.value = selectedAccount.displayInfo.email! ?? this._azureAccountList[0]?.displayInfo?.email! ?? '';
```

Comment:
It looks like the `selectedAccount` is being set to the first item on line 762 above, so do you actually need to do it again here?

(given that you're using `!` that implies that you're saying it should always be defined so it seems like the answer is no...)

Reply:
I can fallback directly to empty string.